docs/content.go

```diff
@@ -850,13 +862,15 @@ Enterprise Performance Computing (EPC)`
 	OciResumeUse   string = `resume <container_ID>`
 	OciResumeShort string = `Resumes all processes previously paused inside the container (root user only)`
 	OciResumeLong  string = `
-  Resume will resume all processes previously paused for the specified container ID.`
+  Resume will resume all processes previously paused for the specified container 
+  ID.`
 	OciResumeExample string = `
   $ singularity oci resume mycontainer`
 
 	OciMountUse   string = `mount <sif_image> <bundle_path>`
-	OciMountShort string = `Mount create an OCI bundle from SIF image (root user only)`
-	OciMountLong  string = `
+	OciMountShort string = `Mount create an OCI bundle from SIF image (root user 
+  only)`
```

Comment:
Do we really need `(root user only)` for every oci command?

Reply:
Yeah.  I think we do.  It looks redundant in the help doc, but remember that only one of these commands will be displayed with any help query.